src/visuals.js

```diff
@@ -119,16 +120,18 @@ export const Visuals = {
     } else {
       this.justPaused = false
     }
-    if (this.frames - this.startingFrame >= this.timer) {
+    if (this.frames - this.startingFrame + FPS >= this.timer) {
```

Comment:
so this waits a full second until the ending starts? Is that just for better rhythm?

Reply:
this ends the game after 1 rather than 0.

3, 2, 1, GAME OVER

rather than

3, 2, 1, 0, GAME OVER

not married to it